### DISCUSSION TITLE
为什么绳匠们都不好好发图片

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#195] [其他]
图片没上传完就提交了，导致大部分帖子的图片都显示不出来。。
![image](https://github.com/user-attachments/assets/20449475-bfea-441b-9a9c-979a19e46482)

**Comment by sabrin1145** [comment]
是这样的

**Comment by Zhishirur** [comment]
对啊。。。简称魔法速度不行。
